### PR TITLE
REG-1402 Reorganizes and redistributes the directories using the PHP version

### PR DESCRIPTION
**Pull request by jose-guidos**
Reorganizes and redistributes the directories using the PHP version

### CODE REVIEW

**Review by MichaelRussell-Regit — APPROVED** [review]

**Comment by jose-guidos** [comment]
Hi @jcc237 

I've updated the README file.

Please, could you review and approve this PR to merge it into main.

Thanks!

File: Readme.md [hunk 1]
```diff
@@ -1,39 +1,103 @@
-Regit docker images
-===================
+# Regit docker images
 
-Docker images used for setting up development and speeding up CI builds.
+Docker images used for setting up development and speeding up CI builds in the Regit project.
```
Comment (jcc237) [inline Readme.md:3]: ....speeding up CI builds in Regit projects.

File: Readme.md
Comment (jcc237) [inline Readme.md:1]: Doesn't refer to pushing dev image?

**Review by jcc237 — CHANGES_REQUESTED** [review]

File: Readme.md (hunk 1 above)
Comment (jose-guidos) [reply Readme.md]: Done

**Review by jcc237 — APPROVED** [review]

**Pull request merged**